tests/components/vallox/conftest.py

```diff
@@ -0,0 +1,74 @@
+"""Common utilities for Vallox tests."""
+
+from datetime import datetime
+import random
+import string
+from typing import Any
+from unittest.mock import patch
+from uuid import UUID
+
+import pytest
+from vallox_websocket_api.vallox import PROFILE
+
+from homeassistant.components.vallox.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_entry(hass: HomeAssistant) -> MockConfigEntry:
+    """Create mocked Vallox config entry."""
+    vallox_mock_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_HOST: "192.168.100.50",
+            CONF_NAME: "Vallox",
+        },
+    )
+    vallox_mock_entry.add_to_hass(hass)
+
+    return vallox_mock_entry
+
+
+def patch_ha_now(mocked_datetime: datetime):
```

Comment:
Please don't patch this. Use async_fire_time_changed instead

Reply:
Ok 👍 It's a leftover from #66443 that isn't even used any more, so removing it.